### PR TITLE
UI update

### PR DESCRIPTION
**Pull request by jasonb5**
PR for #118 and #139

### CODE REVIEW

File: compute/wps/webapp/src/app/configure/regrid.component.ts [hunk 1]
```diff
@@ -16,25 +22,89 @@ export class RegridModel {
...
+  regridMethods: string[] = [];
+
+  regridOptions = [
+    {
+      name: 'ESMF',
+      methods: ['Conserve', 'Linear', 'Patch']
+    },
+    {
+      name: 'Regrid2',
+      methods: ['Conserve', 'Linear']
```
Comment (doutriaux1) [inline compute/wps/webapp/src/app/configure/regrid.component.ts]: there's no method for regrid2, just use "area_weighted"

**Review by doutriaux1 — CHANGES_REQUESTED** [review]
@jasonb5 looks good, just one detail for regrid2 there is no "regrid_method", only one and it is "Area weighted", otherwise all looks ggod.

**Review by doutriaux1 — APPROVED** [review]
make sure it still works 😉 but i think it's ignored by cdms

**Comment by jasonb5** [comment]
Yea I checked it, you can pass anything to regridMethod when regridTool is set to "regrid2".

**Pull request merged**